bundles/ui/org.eclipse.smarthome.ui/src/main/java/org/eclipse/smarthome/ui/internal/items/ItemUIRegistryImpl.java

```diff
@@ -173,6 +174,9 @@ public Widget getWidget(String itemName) {
         for (ItemUIProvider provider : itemUIProviders) {
             Widget currentWidget = provider.getWidget(itemName);
             if (currentWidget != null) {
+                if (currentWidget instanceof Default) {
```

Comment:
Okay, the function is different then I mean, I think we return the fallback if it is no instance of...